hasjob/views/listing.py

```diff
@@ -560,37 +560,65 @@ def pinnedjob(domain, hashid):
 @app.route('/reject/<hashid>', defaults={'domain': None}, methods=('GET', 'POST'))
 @lastuser.requires_permission('siteadmin')
 def rejectjob(domain, hashid):
+    def send_reject_mail(request, reject_type, post):
+        if reject_type == 'reject':
+            subject = "About your job post on Hasjob"
+            template = 'reject_email.html'
+        elif reject_type == 'ban':
+            subject = "About your account and job posts on Hasjob"
+            template = 'reject_domain_email.html'
+        else:
+            return
```

Comment:
I just wanted to put the mail sending part(that's below this subject and template deciding section) in a function. If the mail sending part is in the if blocks, no need for the dictionary really.